### ISSUE TITLE
No entries in thaumonomicon page

### ISSUE THREAD

**Issue by MitchP404** [phantamanta44/tinkers-evolution#84]
I added this mod to my server and immediately realized that I wanted primal metal armor. However, the thaumonomicon page for this mod is completely black. I have tried to scan all of the tinkers stuff, a primordial nodule (haven't gotten a full pearl yet), and void seeds but still can't get any entries. I looked in the code to see if I could find any hints on what I am supposed to scan, and all I could find was something about a "tconevo.tool_table" but I can't figure out what item this is. I already tried scanning the tool station from tinker's but that didn't work. Unless I am missing something, I'm starting to think this might be a glitch, maybe something to do with the fact that I added it in after I scanned tinkers blocks.

**Comment by phantamanta44** [comment]
yeah, this is a known bug. for now, you can use cheats to get the research. sorry about that

**Comment by MitchP404** [comment]
No problem, thanks for the response! Is there a command to specifically get that research?

**Comment by phantamanta44** [comment]
there's some subcommand for thaumcraft's `/thaum` command, but i don't remember the exact syntax

**Comment by MitchP404** [comment]
Tried everything I could with /thaum, couldn't figure anything out, however I am getting into modding so I decided to take another look at the code. It looks like TCONEVO_BASE in tconevo.json is missing a parent, which, from what I can tell by looking at the Thaumcraft API and something similar from Thaumic Augmentation, is required to unhide the node in the thaumonomicon. Unless this is intentional? If needed, I could fork it and see if I can mess with it at all and test this theory, with your permission of course.
